dtfs-central-api/src/services/repositories/utilisation-reports-repo/utilisation-reports-repo.ts

```diff
@@ -73,8 +73,9 @@ const getUtilisationReportDetailsFilterFromOptions = (options?: GetUtilisationRe
   if (options.reportPeriod) {
     utilisationReportDetailsFilter.reportPeriod = { $eq: options.reportPeriod };
   }
-  if (options.reportStatuses) {
-    utilisationReportDetailsFilter.status = { $in: options.reportStatuses };
+  if (options.excludeNotUploaded && options.excludeNotUploaded === 'true') {
```

Comment:
I think this can just be
```
if (options?.excludeNotUploaded === 'true') {
```
or
```
if (options?.excludeNotUploaded) {
```
if we convert the param to a boolean.

Reply:
We don't need the optional chaining as the function already checks whether or not `options` is defined, but yes :)